reddit_liveupdate/templates/liveupdateeventapp.html

```diff
@@ -37,23 +42,19 @@ <h1 id="liveupdate-title">${thing.event.title}</h1>
 <div class="main-content">
 % if thing.show_sidebar:
 <aside class="sidebar side md-container">
-  % if thing.event.description:
-  <section id="liveupdate-description">
-    ${utils.md(thing.event.description, wrap=True)}
-  </section>
-  % endif
+  <a class="sidebar-expand" href="javascript: void 0;" data-toggle="${_("less about this live thread")}">${_("more about this live thread")}</a>
```

Comment:
This is a fine way to do it, but just for future knowledge I wanted to point out that our JS build process allows you to do translated strings straight up in the JS as well, it looks like:

```javascript
// simple translated string
r._("some text")

// plurals
r.N_('%(comment)d comment', '%(comment)d comments', numComments).format({comment: numComments})
```

There should be a number of examples in the liveupdate JS if you want to see it in action.